pages/place/routers/confirm.js

```diff
@@ -71,11 +71,11 @@ module.exports = settings => {
       if (req.session.form && req.session.form[req.model.id]) {
         return next();
       }
-      return res.redirect(req.buildRoute(settings.page));
+      return res.redirect(req.buildRoute('place.delete'));
     },
     editAnswers: (req, res, next) => {
       delete req.session.form[req.model.id].validationErrors;
-      return res.redirect(req.buildRoute(settings.page));
+      return res.redirect(req.buildRoute('place.delete'));
```

Comment:
This feels too specifically "delete" for a router which is not specifically "delete".

Reply:
Good point - will find a proper solution